test/hmactest.c

```diff
@@ -43,23 +43,23 @@ static struct test_st {
         "", 0, "More text test vectors to stuff up EBCDIC machines :-)", 54,
         "e9139d1e6ee064ef8cf514fc7dc83e86",
     },
-    {
-        "\x0b\x0b\x0b\x0b\x0b\x0b\x0b\x0b\x0b\x0b\x0b\x0b\x0b\x0b\x0b\x0b",
-        16, "Hi There", 8,
+    { { 0x0b, 0x0b, 0x0b, 0x0b, 0x0b, 0x0b, 0x0b, 0x0b,
+        0x0b, 0x0b, 0x0b, 0x0b, 0x0b, 0x0b, 0x0b, 0x0b }, 16,
```

Comment:
I think this would look more like the other entries:
```suggestion
    {
        {
            0x0b, 0x0b, 0x0b, 0x0b, 0x0b, 0x0b, 0x0b, 0x0b,
            0x0b, 0x0b, 0x0b, 0x0b, 0x0b, 0x0b, 0x0b, 0x0b
        }, 16,
```